vsc_pdf_template_transformer/lib/models/tpl_repeater.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:pdf/widgets.dart';
+import 'package:vsc_pdf_template_transformer/utils/evaluator.dart';
+import 'package:vsc_pdf_template_transformer/utils/widget_json_converter.dart';
+import '../utils/widget_builder.dart' as wb;
+
+part 'tpl_repeater.g.dart';
+
+@JsonSerializable(
+  checked: true,
+  disallowUnrecognizedKeys: false,
+  explicitToJson: true,
+)
+class TplRepeater {
+  String className = 'TplRepeater';
+
+  @WidgetJsonConverter()
+  List<dynamic>? array;
+  @WidgetJsonConverter()
+  wb.WidgetBuilder? childTemplate;
+
+  TplRepeater();
+
+  factory TplRepeater.fromJson(Map<String, dynamic> json) =>
+      _$TplRepeaterFromJson(json);
+
+  Map<String, dynamic> toJson() => _$TplRepeaterToJson(this);
+
+  List<Widget> toPdf(Map<String, dynamic> data) {
```

Comment:
I added an invoice template to test this: `vsc_pdf_template_editor/example/assets/test/invoice_tpl_test.json`. Please use it to test the repeater.